home/models.py

```diff
@@ -939,14 +899,14 @@ def get_all_links(self):
 
     def save_revision(
```

Comment:
Can we just remove this method completely? It seems like we're just calling `super` and returning?